packages/types/src/primitive/Type.ts

```diff
@@ -57,8 +57,8 @@ export default class Type extends Text {
       Type._alias('Compact<Index>', 'IndexCompact'),
       // alias Vec<u8> -> Bytes
       Type._alias('Vec<u8>', 'Bytes'),
-      // alias &[u8] -> Bytes
-      Type._alias('&\\[u8\\]', 'Bytes'),
+      // alias &[u8] -> Data (this has no length-prefix)
+      Type._alias('&\\[u8\\]', 'Data'),
```

Comment:
Why not just remove Data type and use U8a everywhere? Is it a rust type?

Reply:
U8a is not exported, I tried that first. And since we already use Data to indicate exactly those, just kept it at `Data` as to not have more sprawl.

Reply:
Ok